src/lib/Components/Lists/SavedShowItemRow.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, color, Flex, Sans, Separator } from "@artsy/palette"
+import { SavedShowItemRowMutation } from "__generated__/SavedShowItemRowMutation.graphql"
+import OpaqueImageView from "lib/Components/OpaqueImageView"
+import Switchboard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { TouchableWithoutFeedback } from "react-native"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+
+interface Props {
+  __id: string
+  _id: string
+  id: string
+  href: string
+  name: string
+  status: string
+  images: {
+    url: string
+  }
+  partner: {
+    name: string
+  }
+  is_followed: boolean
+  relayEnvironment: RelayProp
+}
```

Comment:
I feel like this should be generated for you somewhere?

Reply:
I think it's a nested part of the type generated by a fragment, so I don't think it's accessible from a "this object is this shape" perspective. 

A better approach could be to define the fragment in `SavedFairItemRow` and then spread across that fragment in the `SavedFairs` component instead. I'm working in this code for analytics, so I can take on that refactor after this PR is merged.

Reply:
Yup that'd be preferable indeed. sounds good @ashfurrow